node/resolvers/Mutations/Users.ts

```diff
@@ -180,6 +180,17 @@ export const addUser = async (_: any, params: any, ctx: Context) => {
   } = ctx
 
   try {
+    const costCenter = await ctx.clients.organizations.getCostCenterById(
+      params.costId
+    )
+
+    if (
```

Comment:
```suggestion
    // before adding an user to a cost center we check if the cost
    // center exists and if it has a valid name, otherwise both
    // login and UI might break.
    if (
```